### DISCUSSION TITLE
Supported device TuYa TS0601_switch is not supported

### DISCUSSION THREAD

**Discussion by losip** [Koenkk/zigbee2mqtt#11658] [Devices]
Once again, I have bought something from the supported devices list : https://www.zigbee2mqtt.io/devices/TS0601_switch.html only to find when they arrived that Z2M reports them as unsupported, I suspect Tuya at their tricks again changing the manufacturer code - this time to _TZE200_7deq70b8.

I wish I knew how to 'work' github so I could add this to the Tuya.js file. It can't be difficult.

I wonder if anyone can either do this for me or list the actions that must be taken for a github newbie?  For instance, what is a "pull request"? The terminology is unknown to me. Or, point me at a "Github for Dummies" guide?

Thanks

**Comment by losip** [comment]
The above manufacturer code is for the double pole switch; the single switch with a manufacture code of _TZE200_js3mgbjb is supported in the Edge version and I guess will filter thru to prod version

**Reply by losip** [reply]
I feel so helpless: I can see I just need to add: 
            {modelID: 'TS0601', manufacturerName: '_TZE200_7deq70b8'},
to the existing fingerprints but I don't know how

**Reply by Koenkk** [reply]
Added.

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

**Reply by losip** [reply]
Many thanks. I'm already on edge so I'll update soon

**Reply by losip** [reply]
It works! Thanks for your quick attention @Koenkk
